src/aggregation/README.md

```diff
@@ -0,0 +1,27 @@
+# Code Organization
+
+The code is organized in submodules:
```

Comment:
Can we add a link to elasticsearch doc somewhere and explain that this aggregation collector has been designed to mimick its behavior?

Reply:
yep good idea